debian/marquee/nginx/sites/whatwg.org.conf

```diff
@@ -0,0 +1,196 @@
+server {
+    server_name whatwg.org;
+    root /var/www/whatwg.org;
+
+    ssl_certificate /etc/letsencrypt/live/whatwg.org/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/whatwg.org/privkey.pem;
+
+    include /etc/nginx/whatwg.conf;
+
+    error_page 404 /404;
+    error_page 410 /410;
+
+    # There are many files with no extension, and a lot of default_type
+    # directives to get it right. The default is text/html.
+    default_type text/html;
+    charset utf-8;
```

Comment:
charset utf-8;

We should be setting this for all hosts, in `debian/marquee/nginx/conf/whatwg.conf`.

Right now, because we’re not, all our .html, .js, .css content is being served with `Content-Type` headers that have no `charset` param.

Reply:
charset utf-8;
>
> We should be setting this for all hosts, in `debian/marquee/nginx/conf/whatwg.conf`.
>
> Right now, because we’re not, all our .html, .js, .css content is being served with `Content-Type` headers that have no `charset` param.

I raised https://github.com/whatwg/misc-server/pull/26 for that — and also to ensure we serve `.js` files as `text/javascript` (instead of the nginx default of `application/javascript`.  

And for that see the related issue at https://trac.nginx.org/nginx/ticket/1407

Reply:
That's now fixed, so I'll delete it here. Thanks!